app/src/main/java/org/ieeeguc/ieeeguc/models/Committee.java

```diff
@@ -92,8 +93,79 @@ public void onResponse(Call call, Response response){
         });
 
     }
+    /**
+     * This Method is used to create a new committee
+     * @param {string} token [token of the logged in user]
+     * @param {string} name [name of the committee]
+     * @param {string} description [description of the committee]
+     * @param {HttpResponse} HTTP_RESPONSE [http interface instance which is the response coming from the server after creating the new committee]
+     * @return {void}
+     */
+
+    public static void addCommittee(String token,String name,String description,final HTTPResponse HTTP_RESPONSE) {
+        OkHttpClient client = new OkHttpClient();
+        JSONObject j = new JSONObject();
+        try {
+            j.put("name", name);
+            j.put("description", description);
+            RequestBody body = RequestBody.create(CONTENT_TYPE, j.toString());
+            Request request = new Request.Builder()
+                    .url("http://ieeeguc.org/api/committee")
+                    .header("user_agent", "Android")
+                    .header("Authorization", token)
+                    .post(body)
+                    .build();
+            client.newCall(request).enqueue(new Callback() {
+                @Override
+                public void onFailure(Call call, IOException e) {
+                    MainActivity.UIHandler.post(new Runnable() {
+                        @Override
+                        public void run() {
+                    HTTP_RESPONSE.onFailure(-1, null);
+                        }});
+                    call.cancel();
+                }
+
+                @Override
+                public void onResponse(Call call, Response response) {
+
+                    final int code = response.code();
+                    String body = null;
```

Comment:
Why do you assign the body as null use `String body = response.body().string();' and don't forget to catch `Exception` instead of `JSONException`